includes/Generator/Product.php

```diff
@@ -337,6 +337,7 @@ protected static function generate_variable_product() {
 				'attributes'        => $possible_attribute,
 				'regular_price'     => $price,
 				'sale_price'        => $sale_price,
+				'cogs_value'        => round($price * (1 - self::$faker->numberBetween(15, 60) / 100), 2),
```

Comment:
```suggestion
				'cogs_value'        => round( $price * ( 1 - self::$faker->numberBetween( 15, 60 ) / 100 ), 2 ),
```
Just some formatting